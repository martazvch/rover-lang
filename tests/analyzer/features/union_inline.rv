fn main() {
    var a: int|bool
    var b: bool|int = 1
    b = true
    a = b
}

