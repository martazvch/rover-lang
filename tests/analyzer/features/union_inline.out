[Function declaration main]
    [Variable declaration index: 0, scope: local]
        [Null]
    [Variable declaration index: 1, scope: local]
        [Int 1]
    [Bool true]
    [Assignment index: 1, scope: local]
    [Variable index: 1, scope: local]
    [Assignment index: 0, scope: local]
